source/dbo/Stored Procedures/Data Migration/uspMigrationSpecimensMatchesToReview.sql

```diff
@@ -1,13 +1,16 @@
 ﻿CREATE PROCEDURE [dbo].[uspMirgationSpecimensMatchesToReview]
 	@MigrationRun INT = NULL
 AS
-	SELECT 
-		[MigrationNotificationId]			AS 'MigrationNotificationId' 
-		--,[NotificationDate]					AS 'NotificationDate' 
-		--,DATEPART(YEAR, NotificationDate)	AS 'NotificationYear'
-		,[LegacyETSId]						AS 'EtsId'
-		,d.MigrationNotes					AS 'Comments'
- 
-		FROM [dbo].[MigrationRunResults] mrr
-		INNER JOIN [dbo].[vwMigrationDubiousSpecimenMatches] d on d.EtsId = mrr.LegacyETSId
-		WHERE MigrationRunId = @MigrationRun
+	SELECT
+		mrr.MigrationNotificationId							AS 'MigrationNotificationId',
+		mrr.LegacyETSId										AS 'EtsId',
+		mrr.NTBSNotificationId								AS 'NtbsId',
+		mdsm.ReferenceLaboratoryNumber						AS 'ReferenceLaboratoryNumber',
+		esm.SpecimenDate									AS 'SpecimenDate',
+		mrr.NotificationDate								AS 'NotificationDate',
+		dbo.ufnGetTreatmentEndDate(mrr.NTBSNotificationId)	AS 'TreatmentEndDate',
+		mdsm.MigrationNotes									AS 'MigrationNotes'
+	FROM [dbo].[MigrationRunResults] mrr
+		INNER JOIN [dbo].[MigrationDubiousSpecimenMatches] mdsm ON mdsm.EtsId = mrr.LegacyETSId
```

Comment:
Shouldn't this be vwMigrationDubiousSpecimenMatches?

Reply:
`vwMigrationDubiousSpecimenMatches` aggregates `MigrationDubiousSpecimenMatches` by notification, so we need to go to the latter since we want one row per specimen rather than one row per notification.